wemod_launcher/wemod

```diff
@@ -9,7 +9,7 @@ import subprocess
 from typing import Optional
 
```

Comment:
moving this file may break backwards compat,
well i guess its fiine but i just want to reduce the amount of work,
if we change it all then the users will all report nothing happens,
something to think on.
Realy keep in mind that on windows wemod is as simple as installing one file,
therefore the idea that most users are unexperienced is not far off,
just keep that in mind.

Reply:
Well a post might work, 
well the thing is i have not done anything related to social media,
so i don't have any idea whats going on there.
Well i do use social media but not anything related to the wemod-launcher.  
Just so you know that this is something you might have to look for yourself.